pkg/monitoring/monitoring.go

```diff
@@ -218,7 +224,22 @@ func OverwriteAppAnswers(rawAnswers map[string]string, annotations map[string]st
 		rawAnswers[key] = value
 	}
 
-	return rawAnswers
+	catalogID, err := GetMonitoringCatalogID(version, catalogTemplateClient)
+	if err != nil {
+		return rawAnswers, catalogID, err
+	}
+	return rawAnswers, catalogID, nil
+}
+
+func GetMonitoringCatalogID(version string, catalogTemplateClient mgmtv3.CatalogTemplateInterface) (string, error) {
+	if version == "" {
+		template, err := catalogTemplateClient.GetNamespaced(catalogTemplatesNamespace, rancherMonitoringTemplateName, metav1.GetOptions{})
```

Comment:
any reason we can't use lister here? We should avoid doing the direct db calls if possible